MAUI/TimePicker/customizations.md

```diff
@@ -0,0 +1,403 @@
+---
+layout: post
+title: Customizations in .NET MAUI Time Picker Control | Syncfusion
+description: Learn about customizations in Syncfusion .NET MAUI Time Picker (SfTimePicker) control and its basic features.
+platform: maui
+control: SfTimePicker
+documentation: ug
+---
+
+# Customizations in .NET MAUI Time Picker (SfTimePicker)
+
+The [.NET MAUI Time Picker] header, column header, footer and selection views can be customized.
+
+## Header customization
+
+You can customize the time picker header by using the `HeaderView` property of `SfTimePicker`.
+
+### Set header text
+
+The SfTimePicker control allows you to add the header text by setting the [Text] property in the [PickerHeaderView]. To enable the header view by setting the [Height] property grater than 0 of [PickerHeaderView]. The default value of `Height` property is 0.
```

Comment:
The SfTimePicker control allows you to add the header text by setting the [Text] property in the [PickerHeaderView]. To enable the header view, set the [Height] property of the [PickerHeaderView] to a value greater than 0. The default value of the `Height` property is 0.

Reply:
Comments modified.